Launch orbit

chris-perts opened the pull request:
@BSauve983 this melding of the codes worked pretty well. I still don't know what to do about the trigger-based approach versus the wait-based approach. But this got the test craft into orbit!

cmacrander committed:
simple minification (we haz tu much codez!)

from command line in script directory: minify_ks.py foo.ks foo_min.ks

cmacrander committed:
incorporating orbit into launch

cmacrander committed:
readme update

cmacrander commented:
Also, don't know how I wound up in my weird PERTS-centric github account. Sorry about that.

BSauve983 commented on README.md:
```diff
@@ -15,3 +15,11 @@ Code suggestions from [the kOS design patterns page](http://ksp-kos.github.io/KO
++ Chris writes excessively verbose code, and his files maxed out the probe's storage.
+    - You can use minify_ks.py to do a little bit of space saving. It works a whole lot better than `COMPILE`.
+    - But you have to `RENAME` all the minified files back to their original names once they're on your craft's CPU or else the imports won't work.
+    - This is generally super annoying.
```
The are other strategies to deal with this per [this kOS page](http://ksp-kos.github.io/KOS_DOC/general/volumes.html):
- Parts farther along the tech tree have more storage.
- You can pay for additional storage volume during vehicle construction.
- You can store programs in the "archive" (volume number 0), which has unlimited space, and run from there.  This can only be done at a range of 100km, unless you have to have enough antennae, but they aren't specific about the requirements.
- You can put multiple units on the spacecraft, which are automatically "networked".  Effectively, you use them by switching between volumes with `SWITCH TO #`.

BSauve983 commented on minify_ks.py:
```diff
@@ -0,0 +1,32 @@
+        print 'Bad arguments. minify_ks.py <inputfile> <outputfile>'
+        sys.exit(2)
+
+    minify(sys.argv[1], sys.argv[2])
```
But then you gotz to have Python installed on your computer and know how to use it!  I CAN'T LEARN NEW THINGS!

cmacrander committed:
Local declarations in launch, added launch inclination.

East = 90

cmacrander committed:
more useful minification

reduces copy commands needed at launch; removal of `run` references
reduces file naming complications.
